src/System.Management.Automation/engine/LanguagePrimitives.cs

```diff
@@ -593,7 +593,9 @@ public static PSDataCollection<PSObject> GetPSDataCollection(object inputValue)
         /// <param name="second">Object to compare first to.</param>
```

Comment:
@daxian-dbw, your last commit had 1 failures in `PowerShell-CI-static-analysis`
Verify Markdown Links.Verify links in /home/vsts/work/1/s/CHANGELOG/6.0.md.https://msdn.microsoft.com/powershell/reference/6/about/about_jobs should work
```powershell
Failed to complete request to "https://msdn.microsoft.com/powershell/reference/6/about/about_jobs". Response status code does not indicate success: 404 (Not Found).
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 119
119:                                     throw "Failed to complete request to `"$url`". $($_.Exception.Message)"

```

Reply:
This will be addressed by #11653